src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -93,6 +155,7 @@ protected override void Dispose(bool disposing)
             state.CancellationToken = cancellationToken;
             state.RequestMessage = request;
             state.Handler = this;
+            state.ProxyPolicy = _proxyPolicy;
```

Comment:
> This code is still wrong regarding when to return FALSE for UseProxy:

sorry, is there some mistake in this fragment too? (line 93-95 old code and 155-158, new code)